pkg/operator/configobservation/scheduler/observe_scheduler_test.go

```diff
@@ -23,14 +23,22 @@ func TestObserveSchedulerConfig(t *testing.T) {
 	tests := []struct {
 		description   string
 		configMapName string
+		found         bool
```

Comment:
nit: perhaps changing the name to appropriate variable name like `updateNeeded`?

Reply:
updated